src/kirby/components/item/item.component.integration.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { SpectatorHost, createHostFactory } from '@ngneat/spectator';
+import { IonicModule } from '@ionic/angular';
+
+import { TestHelper } from '../../testing/test-helper';
+import { DesignTokenHelper } from '../../helpers/design-token-helper';
+import {
+  ListComponent,
+  InfiniteScrollDirective,
+  ListItemTemplateDirective,
+} from '@kirbydesign/designsystem/list';
+import { ItemComponent } from './item.component';
+import { SpinnerComponent, IconComponent, CardComponent } from '@kirbydesign/designsystem';
+import { ListItemColorDirective } from '../list/directives/list-item-color.directive';
+
+const size = DesignTokenHelper.size;
+
+describe('ItemComponent in Kirby List', () => {
+  let ionList: HTMLElement;
+  let itemsInList: HTMLElement[];
+
+  let spectator: SpectatorHost<ListComponent>;
+  const createHost = createHostFactory({
+    component: ListComponent,
+    imports: [IonicModule.forRoot()],
+    declarations: [
+      ItemComponent,
+      ListComponent,
+      SpinnerComponent,
+      IconComponent,
+      CardComponent,
+      InfiniteScrollDirective,
+      ListItemColorDirective,
+      ListItemTemplateDirective,
+    ],
+  });
+
+  describe('inside list', () => {
+    beforeEach(async () => {
+      spectator = createHost<ListComponent>(
+        `
+        <kirby-list [items]="[{ name: 'Item1' }, { name: 'Item2' }, { name: 'Item3' }]">
+          <kirby-item *kirbyListItemTemplate="let item">{{ item.name }}</kirby-item>
+        </kirby-list>
+        `
+      );
+      ionList = spectator.queryHost('ion-list');
+      await TestHelper.whenHydrated(ionList);
+      itemsInList = spectator.queryAll('ion-list ion-item');
+    });
+
+    it('should create list wrapper', () => {
+      expect(spectator).toBeTruthy();
+    });
+
+    it('should items in list', () => {
+      expect(itemsInList).toBeTruthy();
+    });
+
+    it(`should render first with padding top ${size(
```

Comment:
Could be simplified even further:
```suggestion
    it(`should render first and last item with correct padding`, 
```